src/pages/AdminSettings/AdminAccess/AdminAccessTable/useAdminAccessList.ts

```diff
@@ -0,0 +1,64 @@
+import { useInfiniteQuery } from '@tanstack/react-query'
+import { useParams } from 'react-router-dom'
+import { z } from 'zod'
+
+import Api from 'shared/api'
+import { NetworkErrorObject } from 'shared/api/helpers'
+
+const RequestSchema = z.object({
+  count: z.number(),
+  next: z.string().nullable(),
+  previous: z.string().nullable(),
+  results: z.array(
+    z.object({
+      ownerid: z.number(),
+      username: z.string().nullable(),
+      email: z.string().nullable(),
+      name: z.string().nullable(),
+      isAdmin: z.boolean(),
+      activated: z.boolean(),
+    })
+  ),
+  totalPages: z.number(),
+})
+
+type URLParams = {
+  provider: string
+}
+
+export const useAdminAccessList = () => {
+  const { provider } = useParams<URLParams>()
```

Comment:
If you're making a request to `/internal` and on the admin page you won't need the provider, as it's not actually used if you look at the request in the network tab

Reply:
oooo tyty